Gruntfile.js

```diff
@@ -70,9 +70,9 @@ module.exports = function (grunt) {
                 reporter: reporter,
             };
 
-        // For running in docker/travis
+        // For running in docker
         if (process.env.PUPPETEER_SKIP_DOWNLOAD) {
-            runnerOptions.executablePath = 'google-chrome-unstable';
+            runnerOptions.executablePath = 'google-chrome-stable';
```

Comment:
This sounds like a good change, but just curious how it relates to the switch to `uv`?

Reply:
It doesn't. It simplified the image (depending on stable seemed better than unstable+bunch of fonts), and I think it reduced the size and build time non-trivially (although I didn't take precise measurements). Also, this is what the most recent online sources said about how to use Puppeteer in Docker.